### DISCUSSION TITLE
MIT License in standalone.js?

### DISCUSSION THREAD

**Discussion by Alvie** [scalar/scalar#1335] [Q&A]
Hi!

I want to use this but I am a little confused about the MIT Licensing. From what I understand, is that I need to include the copyright notice and the license with the "software", but I am retrieving it with NPM and only exposing the standalone.js

I am not too familiar with how licenses work, I want to include the copyright notice and the license info, but do I need to do this separately. Perhaps it should be included in the standalone.js instead?
https://github.com/scalar/scalar/discussions?discussions_q=MIT+License+in+standalone.js%3F
Sorry if this is a very nooby question 😅

**Comment by Alvie** [comment]
Reading into bootstrap's guidelines (also using the MIT License), I would need to ensure the copyright notice and the license header stays there, but it isn't there in the first place for @scalar/api-reference (at list the browser/standalone.js file).

Also, please let me know if I would need to include the credits in the front end / GUI of my application somewhere accessible to the end user?

![image](https://github.com/scalar/scalar/assets/8447791/5e3c6ff8-5874-4a89-a07f-79842cccdfed)
![image](https://github.com/scalar/scalar/assets/8447791/eae98a86-b533-4698-a019-e43022af47c1)

**Comment by marclave** [comment]
we definitely need to add it! We will get that fixed up this week ✨ I opened an issue

https://github.com/scalar/scalar/issues/1482
